spynnaker/pyNN/models/common/neuron_recorder.py

```diff
@@ -784,7 +888,8 @@ def get_buffered_sdram_per_record(self, variable, vertex_slice):
                 return 0
             size = self.__per_timestep_datatypes[variable].size
             return self._N_BYTES_FOR_TIMESTAMP + size
-        n_neurons = self._count_recording_per_slice(variable, vertex_slice)
+        n_neurons = self._count_recording_per_slice(
+            variable, vertex_slice, max_rewires_per_ts)
```

Comment:
I think this will not end up being needed here if not being treated as a normal variable